Client/Frontend/Browser/New Tab Page/Sections/NTPDefaultBrowserCalloutProvider.swift

```diff
@@ -43,7 +43,10 @@ class NTPDefaultBrowserCalloutProvider: NSObject, NTPObservableSectionProvider {
                         layout collectionViewLayout: UICollectionViewLayout,
                         sizeForItemAt indexPath: IndexPath) -> CGSize {
         
-        return fittingSizeForCollectionView(collectionView, section: indexPath.section)
+        var size = fittingSizeForCollectionView(collectionView, section: indexPath.section)
+        size.height = 54
```

Comment:
This will break the height on other layouts like iPad where the text wont wrap as much, no? We should probably find a better way to fix this if possible

Reply:
I can also disable drap and drop when the default browser callout is presented.